src/Core/Checkout/Cart/SalesChannel/CartService.php

```diff
@@ -76,6 +76,21 @@ public function getCart(
         return $this->cart[$cart->getToken()] = $cart;
     }
 
+    public function load(
```

Comment:
where is this used and needed?

Reply:
hey! sorry, It should be a draft so far - and it will be used in the store-api route